doc/examples/reconst_fwdti.py

```diff
@@ -0,0 +1,185 @@
+"""
+==========================================================================
+Using the free water elimination model to remove free water contamination
+==========================================================================
+
+As shown previously (see :ref:`example_reconst_dti`), the diffusion tensor
+model is a simple way to characterize the diffusion anisotropy. However, this
+model is not specific to particular type of tissue. For example, diffusion
```

Comment:
"to particular type of tissue" => "to a particular type of tissue" or "to particular types of tissue". 

Also, this fragment is not so clear, so I think that you can jump straight to the next clause: 

    However, in regions near the cerebral ventricles and parenchyma ...